### PR TITLE
Maven Dependency Cleanup - Appengine

### PR DESCRIPTION
**Pull request by kurtisvg**
This PR cleans up the dependency structure for the Appengine folder. It does the following:

1. Changes parents for individual projects to our `shared-configuration` so they users can move a relevant folder without breaking things.
2. Addresses any exists check-style violations 
3. Adds the `Open in Cloud Shell` button for all README.md

Additionally, it deletes the appengine/channel samples since the service has been shutdown.

### CODE REVIEW

**Comment by lesv** [comment]
Wow - you just want me unhappy. 241 files?

File: appengine/remote/remote-server/src/main/java/com/example/appengine/remote/RemoteServlet.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
-/**
- * Copyright 2016 Google Inc.
+/*
+ * Copyright 2015 Google Inc.
```
Comment (lesv) [inline appengine/remote/remote-server/src/main/java/com/example/appengine/remote/RemoteServlet.java]: In general (and I saw this earlier), we don't change the dates.  (Rolling them forward or backward).

**Comment by lesv** [comment]
I presume you meant Open in Cloud Shell, not SQL.

**Comment by kurtisvg** [comment]
Merging - Issue with test is because of Vision Beta.

**Pull request merged**